charmhub/info.go

```diff
@@ -37,5 +37,6 @@ func (c *InfoClient) Info(ctx context.Context, name string) (transport.InfoRespo
 	if err := c.client.Get(ctx, path, &resp); err != nil {
 		return resp, errors.Trace(err)
 	}
+
```

Comment:
Normally a nit, I wouldn't comment on, but seem to be the only change in this file.